string last_state_fen_string
---
int8 return_code
string movement_in_fen